components/WalletLayout.vue

```diff
@@ -102,7 +91,16 @@ export default {
 </script>
 
 <style scoped>
-.s-page {
-  background: rgb(240, 240, 240);
+.q-layout__section--marginal {
+  /* Hides line of primary color on mobile */
+  background-color: transparent;
+}
+.bg-gradient {
+  /* fallback for old browsers */
+  background: #FFFFFF;
+  /* Chrome 10-25, Safari 5.1-6 */
```

Comment:
I doubt we support "Chrome 10-25, Safari 5.1-6", but I do think keeping the even more basic white background in the list is important.